### ISSUE TITLE
code processing time is large 

### ISSUE THREAD

**Issue by shivanshsoni** [typetools/checker-framework#1186]
I wrote a 7 line code and in cmd it took around 10sec to show output which is very large as compared to javac. This must be solved. As for large codes it will take too  much time to process them .

**Comment by mernst** [comment]
I agree this is a problem.  There are some concrete ideas for how to improve the run time at https://checkerframework.org/gsoc-2017-ideas.html#performance .
